fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalCatalog.java

```diff
@@ -44,6 +45,9 @@ public class EsExternalCatalog extends ExternalCatalog {
 
     private static final Logger LOG = LogManager.getLogger(EsExternalCatalog.class);
     private EsRestClient esRestClient;
+    private static final List<String> REQUIRED_PROPERTIES = Lists.newArrayList(
+            EsResource.HOSTS
+    );
```

Comment:
Maybe it is better to use `ImmutableList` or `ImmutableSet`？The same goes for other `REQUIRED PROPERTIES`.